proxies/shared/policies/ExtractVariables.NhsAppAccounts.Get.Request.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ This policy takes the ods-organisation-code value from the URI of the request and stores it within the Apigee flow variable data.ods-organisation-code. This value is thus accessible from other policies in the proxy, referencing it with {data.messageId}.
+
+ For more information, see the following resources:
+    * https://docs.apigee.com/api-platform/fundamentals/introduction-flow-variables
+-->
+<ExtractVariables async="false" continueOnError="false" enabled="true" name="ExtractVariables.NhsAppAccounts.Get.Request">
```

Comment:
Maybe one for discussion as I'm not sure what the capabilities of APIGEE are and whether we'd want it, but I'm wondering whether we could leverage APIGEE to enforce that ods-organisation-code is a required query param?

Reply:
Yes that's possible. I'll add it in.

Reply:
Done!